### PR TITLE
Upgrade guide to v1.1

### PR DESCRIPTION
**Pull request by alasarr**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 - Update TypeScript typings to Cloud Native [#147](https://github.com/CartoDB/carto-react/pull/147)
 - Remove SourceTypes from react-api public api [#147](https://github.com/CartoDB/carto-react/pull/147)
+- Add support to Cloud Native SQL API [#150](https://github.com/CartoDB/carto-react/pull/150)
```
Comment (VictorVelarde) [inline CHANGELOG.md:7]: 👍🏻

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
...
 
 ## (prerelease) 1.1.0-alpha.2 (2021-05-28)
 ## (prerelease) 1.1.0-alpha.1 (2021-05-27)
```
Comment (VictorVelarde) [inline CHANGELOG.md:10]: We might probably want to remove the alphas from changelog

File: docs/upgrade-guide.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Upgrade Guide
+
+## Upgrading from @carto/react v1.0 to v1.1
+
+### executeSQL parameters via destructuring 
+
+ExecuteSQL parameters are passed using object destructuring.
```
Comment (VictorVelarde) [inline docs/upgrade-guide.md:7]: Shouldn't we indicate it is a 'breaking' ?

File: docs/upgrade-guide.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+```javascript
+executeSQL({ credentials, query })
+```
+
+### Credentials apiVersion
+
+A new parameters is included in cartoSlice
```
Comment (VictorVelarde) [inline docs/upgrade-guide.md:15]: typo: "a new parameter"

**Review by VictorVelarde — APPROVED** [review]
👍🏻

**Comment by VictorVelarde** [comment]
This needs an update with master first!

File: docs/upgrade-guide.md (hunk 3 above)
Comment (alasarr) [reply docs/upgrade-guide.md:7]: Nope, all in in the upgrade is a breaking

File: CHANGELOG.md (hunk 2 above)
Comment (alasarr) [reply CHANGELOG.md:10]: Yeah

**Pull request merged**